doc/blob.rst

```diff
@@ -170,7 +170,9 @@ Alternatively, you can define a named access policy on the server:
     si = SignedIdentifier()
     si.id = policy_name
     si.access_policy.expiry = '2016-01-01'
-    si.access_policy.permission = BlobSharedAccessPermissions.READ
+    si.access_policy.permission = (
+        BlobSharedAccessPermissions.READ + BlobSharedAccessPermissions.LIST
```

Comment:
This looks good, but switch to ContainerSharedAccessPermissions and update add docs to say this gives not only read, but list permissions. The policy itself is named 'readonly...' and listing truly isn't reading or it would be part of read.